src/main/java/dev/blaauwendraad/masker/json/ByteTrie.java

```diff
@@ -28,21 +33,23 @@
  * <p> And lastly, we can make a small optimization to remember all the distinct lengths of the target keys, so that
  * we can fail fast if the incoming key is not of the same length.
  */
+@ParametersAreNonnullByDefault
```

Comment:
Nit: we do this in all packages now (once https://github.com/Breus/json-masker/pull/50 merges)

Reply:
Done